### PR TITLE
Minor GlassBR improvements

### LINKED ISSUE #833: Product Use Case Table Number in GlassBR SRS
This table is called "Table 2", despite being the first table in the document. It should be updated to "Table 1" to match caseStudies and be correct.

product use

The rest of the tables then need to be shifted ie. the following should be tables 2 and 3:

other tables

### LINKED ISSUE #868: Input Data Constraints Table | Significant Digits and Units
## Significant Digits
- [ ] Change Uncertainties in Drasil to 10%
- [x] ~~Update significant digits in caseStudies - wait until #824~~
- [x] Change TNT typical value to 1.0 in caseStudies
- [x] Change TNT typical value to 1.0 in Drasil

## Units
(Related to discussion in #824) In Drasil, _a_ and _b_ are given in metres, and in millimetres in caseStudies.

sig figs and units

### LINKED ISSUE #876: Intro Differences in Requirements Section
- [x] Add the Drasil (left) Requirements intro to caseStudies
- [x] Add the caseStudies Functional Requirements intro to Drasil
- [x] ~~Update caseStudies for other examples~~ not doing this anymore

req intros

- [x] Update casestudies Nonfunctional Requirement intro to match Drasil (including section title)

non-func req

### LINKED ISSUE #877: Update R1
- [x] Update caseStudies (right) to include a referable table instead of an inline one
- [x] Update the note in Drasil after units have been changed (#824)
- [x] Add a period after "the blast" in Drasil

R1

### PR DESCRIPTION
**Pull request by samm82**
The merge of `glassImprovements` closes #833, closes #868, closes #876, and closes #877 - various smaller issues from last summer.

### CODE REVIEW

**Comment by samm82** [comment]
Not sure why the Travis CI build is failing - works on my machine. Is it because it requires a review? I don't think that would affect the build though.

**Comment by smiths** [comment]
@samm82, the CI runs independently of the review, so that is not the explanation.  Click on the Details link above to see why the build failed.

**Comment by samm82** [comment]
It looks like the issues are related to hlint, which I haven't set up/looked at yet. I'll fix this tomorrow so this can be merged.

**Comment by smiths** [comment]
@Mornix should be able to help with hlint issues, if needed.

File: code/drasil-data/Data/Drasil/SentenceStructures.hs [hunk 1]
```diff
@@ -220,10 +221,12 @@ none :: Sentence
 none = S "--"
 
 found :: Double -> Sentence
-found x = (addPercent . realToFrac) (x*100)
-
+found x
```
Comment (JacquesCarette) [inline code/drasil-data/Data/Drasil/SentenceStructures.hs]: As discussed elsewhere, this hack is worse than the problem it supposedly fixes - I won't merge this in...

**Comment by samm82** [comment]
Sorry - I guess I got carried away with adding changes - "minor" changes can easily get bigger. Should I split this into multiple PRs, or should I just move the commits in question (`glassTypeAbbrs` and `found`) to another branch so this can be merged? @JacquesCarette In future I'll be sure to make smaller PRs 😁

**Comment by samm82** [comment]
@JacquesCarette I made two new branches - one for fixing precision and one for removing the glass attributes string list. Would this make it OK to merge, or should I still split it?

**Pull request merged**